src/main/java/com/example/bcsd/Model/Article.java

```diff
@@ -0,0 +1,66 @@
+package com.example.bcsd.Model;
+
+public class Article {
+    private int articleId;
+    private String author;
+    private int boardId;
+    private String title;
+    private String content;
+    private String writeDateTime;
+    private String modifyDateTime;
+
+    public Article() {}
+
+    public Article(int articleId, String author, int boardId, String title, String content, String writeDateTime) {
+        this.articleId = articleId;
+        this.author = author;
+        this.boardId = boardId;
+        this.title = title;
+        this.content = content;
+        this.writeDateTime = writeDateTime;
+    }
+
+    public Article(int articleId, String author, int boardId,
+                   String title, String content,
+                   String writeDateTime, String modifyDateTime) {
+        this.articleId = articleId;
+        this.author = author;
+        this.boardId = boardId;
+        this.title = title;
+        this.content = content;
+        this.writeDateTime = writeDateTime;
+        this.modifyDateTime = modifyDateTime;
+    }
+
+    public int getArticleId() {return articleId;}
```

Comment:
보통 이런 메서드들도 가독성 좋게 해줍니다! 
```suggestion
    public int getArticleId() {
        return articleId;
    }
```

Reply:
그리고 메서드 사이에 개행 한 번씩 추가해주세요!

Reply:
넵 관련해서도 수정하겠습니다!